x-pack/test/apm_api_integration/tests/transactions/transactions_groups_main_statistics.spec.ts

```diff
@@ -39,6 +39,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
               latencyAggregationType: LatencyAggregationType.avg,
               transactionType: 'request',
               environment: 'ENVIRONMENT_ALL',
+              useDurationSummary: false,
```

Comment:
IMHO this needs tests for `useDurationSummary: true`

Reply:
@dgieselaar I agree, but this is not using Synthtrace but ES Archives. We will have to migrate this test to Synthtrace first. Should we scope it as part of this PR ?

Reply:
Ideally yes, unless we have other API tests that cover this endpoint.